api/src/main/scala/com/rasterfoundry/granary/endpoints/Inputs.scala

```diff
@@ -0,0 +1,11 @@
+package com.rasterfoundry.granary.api.endpoints
+
+import com.rasterfoundry.granary.datamodel.PageRequest
+
+import sttp.tapir._
+
+object Inputs {
+
+  val paginationInput: EndpointInput[PageRequest] =
+    query[Option[Int]]("page").and(query[Option[Int]]("limit")).mapTo(PageRequest.apply _)
```

Comment:
I think maybe `limit` should be a refined type with a lower bound of `0` and an upper bound of whatever we want to limit it to. Passing in a `-1` throws a `500` right now.

Reply:
TIL: `NonNegInt` :sunglasses:

Reply:
`PageRequest`s got refinement, but `PaginatedResponse[T]`s didn't because tapir was super confused about codecs

Reply:
I think that's fine, it's less important for the response since we're not parsing that anywhere